dependencies.yaml

```
@@ -658,3 +713,13 @@ dependencies:
             packages: &cupy_packages_cu11
               - cupy-cuda11x>=13.2.0
           - {matrix: null, packages: *cupy_packages_cu11}
+  depends_on_cugraph_pyg:
+    common:
+      - output_types: conda
+        packages:
+          - cugraph-pyg==25.4.*,>=0.0.0a0
+  depends_on_mkl:
+    common:
+      - output_types: conda
+        packages:
+          - mkl<2024.1.0
```

```suggestion
  depends_on_mkl:
    common:
      - output_types: conda
        packages:
          - mkl<2024.1.0
```

Since this is only used in one place, can you please add it to the existing `test_python_pylibwholegraph:` group instead of as a new `depends_on:` entry?

Context: https://github.com/rapidsai/cudf/pull/18014#discussion_r1956411255

---

Ok fair enough, we can leave it as you have it.